<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetAlgorithm</class>
 <widget class="QDialog" name="SetAlgorithm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetAlgorithm</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>1图片</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>2常见参数</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>3特定参数</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>白</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>黑</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>数值：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>391</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>4 选择算法</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>初始化</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>算法选择</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>保存算法</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>读取算法</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>显示结果</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>读取图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>5 结果</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>SetAlgorithm</receiver>
   <slot>sintial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>SetAlgorithm</receiver>
   <slot>readhimage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>SetAlgorithm</receiver>
   <slot>gettab(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sintial()</slot>
  <slot>readhimage()</slot>
  <slot>gettab(QTableWidgetItem*)</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
